x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -68,3 +84,72 @@ export function LayerContextMenu(props: VisualizationLayerWidgetProps<State>) {
     </EuiFormRow>
   );
 }
+
+export function XyToolbar(props: VisualizationToolbarProps<State>) {
+  const [open, setOpen] = useState(false);
+  const hasNonBarSeries = props.state?.layers.some(
+    (layer) => layer.seriesType === 'line' || layer.seriesType === 'area'
+  );
+  return (
+    <EuiFlexGroup justifyContent="flexEnd">
+      <EuiFlexItem grow={false}>
+        <EuiPopover
+          panelClassName="lnsXyToolbar__popover"
+          button={
+            <EuiButtonEmpty
+              iconType="arrowDown"
+              iconSide="right"
+              onClick={() => {
+                setOpen(!open);
+              }}
+            >
+              <EuiIcon type="gear" />
+            </EuiButtonEmpty>
+          }
+          isOpen={open}
+          closePopover={() => {
+            setOpen(false);
+          }}
+          anchorPosition="downRight"
+        >
+          <EuiFormRow
+            display="columnCompressed"
+            label={i18n.translate('xpack.lens.xyChart.fittingLabel', {
+              defaultMessage: 'Fitting function',
+            })}
+            helpText={
+              !hasNonBarSeries &&
+              i18n.translate('xpack.lens.xyChart.fittingDisabledHelpText', {
+                defaultMessage: 'This setting only applies to line and area charts.',
+              })
+            }
+          >
+            <EuiSuperSelect
+              compressed
+              disabled={!hasNonBarSeries}
+              options={(Object.entries(fittingFunctionDescriptions) as Array<
+                [FittingFunction, string]
+              >).map(([id, description]) => {
+                return {
+                  value: id,
+                  inputDisplay: (
+                    <>
+                      <strong>{fittingFunctionTitles[id]}</strong>
+                      <EuiText size="s" color="subdued">
+                        <p className="euiTextColor--subdued">{description}</p>
```

Comment:
Duplicate of the EuiText.color.
```suggestion
                        <p>{description}</p>
```